README.md

```diff
@@ -0,0 +1,28 @@
+# Qtile website
+## Vocab
+- `qtile` or `qtile.org`: qtile's website.
+- `qtail`: Hugo theme built using TailwindCSS for qtile.org website.
+
+## Dependencies
+- Hugo - [Hugo installation docs for your OS](https://gohugo.io/installation/)
+- NodeJS
+- NPM
+
+## Setting up qtile.org locally
+- Install Hugo, nodejs and npm on your computer 
+- Fork and clone [qtile.org repository](https://github.com/qtile/qtile.org) locally to your computer
+- Go to `qtail` directory. `cd qtile.org/themes/qtail`
+- Run `npm install` in your terminal. This will install Tailwind CSS and it's dependencies necessary for qtile.org.
+- Go to the main website directory `/qtile.org` and run `hugo server` or `hugo server --disableFastRender`
+- Run `npm run watch-qtail` on one terminal in `themes/qtail` and `hugo server --disableFastRender` in another terminal window/tab for watching and generating file changes on the fly. This is the ideal environment for making changes.
+![qtile-local-env](https://github.com/unsungNovelty/qtile.org/assets/11845908/3ebda6b9-1d22-4685-9cf1-79f5f81b83ef)
```

Comment:
Can you put this screenshot in this repo? Or alternatively just drop it, I don't know that it adds much here.